pkg/reconciler/stats_reporter.go

```diff
@@ -0,0 +1,172 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package reconciler
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+	"knative.dev/pkg/metrics"
+)
+
+const (
+	InstallSuccessCountName    = "install_success_count"
+	InstallSuccessLatencyName  = "install_success_latency"
+	DeploymentReadyCountName   = "deployment_ready_count"
+	DeploymentReadyLatencyName = "deployment_ready_latency"
+)
+
+var (
+	InstallSuccessCountStat = stats.Int64(
+		InstallSuccessCountName,
+		"Number of successful KnativeServing install",
```

Comment:
Is this what we want "install" to mean? The number of installs of the KnativeServing custom resource? Or do we want it to mean the install of knative-serving itself?

Reply:
With operator, we install knative-serving by installing KnativeServing custom resource. Unless the users install knative-serving by themselves, these two numbers should be the same. And what we actually care about, in my opinion, is the usage of operator to install KnativeServing custom resource.